pkg/NAMESPACE

```diff
@@ -97,3 +103,5 @@ export("jsonFile",
        "uncacheTable")
 
 export("sparkRSQL.init")
+export("print.structType",
```

Comment:
AFAIK `S3method(print, structType)` is the same as `export(print.structType)` -- This is because the way S3 methods are dispatched is `method.type()` is called for `method(type)`